### DISCUSSION TITLE
How to change the modals title on createOptionForm from BelongsToSelect?

### DISCUSSION THREAD

**Discussion by otavio-araujo** [filamentphp/filament#2680] [General]
How to change the modals heading on createOptionForm from BelongsToSelect?

use Filament\Forms\Components\BelongsToSelect;
use Illuminate\Database\Eloquent\Model;
 
BelongsToSelect::make('authorId')
    ->relationship('author', 'name')
    ->createOptionForm([
        Forms\Components\TextInput::make('name')
            ->required(),
        Forms\Components\TextInput::make('email')
            ->required()
            ->email(),
    ]),

**Comment by danharrin** [comment]
`->createOptionAction(fn ($action) => $action->modalHeading('...'))`

**Reply by otavio-araujo** [reply]
Thanks!
Worked just fine!
